src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/ClientRepository.cs

```diff
@@ -208,6 +224,19 @@ public async Task DeleteClientAsync(Guid clientId)
             }
         }
 
+        public async Task DeleteMemberClientsWithThresholdAsync(Guid memberId, int inactiveHoursThreshold)
```

Comment:
Please write test for this repository method.

Reply:
done